src/index.js

```diff
@@ -28,7 +28,24 @@ export async function run() {
       await execShellCommand(optionalSudoPrefix + 'apt-get update');
       await execShellCommand(optionalSudoPrefix + 'apt-get install -y openssh-client xz-utils');
 
-      const tmateReleaseTar = await tc.downloadTool(`https://github.com/tmate-io/tmate/releases/download/${TMATE_LINUX_VERSION}/tmate-${TMATE_LINUX_VERSION}-static-linux-amd64.tar.xz`);
+      // Possible os.arch() values documented here:
+      // https://nodejs.org/api/os.html#os_os_arch
+      // Available tmate binaries listed here:
+      // https://github.com/tmate-io/tmate/releases/
+      let tmateArch;
+      switch (os.arch()) {
+        case 'arm64':
+          tmateArch = 'arm64v8';
+          break;
+        case 'x64':
+          tmateArch = 'amd64';
+          break;
+        default:
+          throw new Error(`Unsupported architecture: ${os.arch()}`)
+          break;
+      }
```

Comment:
Thank you for your contribution!

About the patch: Personally, I like a map approach better (`{arm64: "arm64v8", x64: "amd64"}`).

What is your take, @mxschmitt ?

Also: could you add an axis to the manual test matrix?

Finally, it seems that there are merge conflicts. It would be good to rebase to current `main`.

Reply:
A map is much cleaner-looking.  Thanks for the advice!

Unfortunately, I can't add an arm64 axis to the test matrix.  GitHub Actions does not offer any arm64 VMs.  I'm adding a self-hosted arm64 runner to some of my projects, which is how I discovered the need for this PR.

Reply:
> 
> 
> Unfortunately, I can't add an arm64 axis to the test matrix. GitHub Actions does not offer any arm64 VMs. I'm adding a self-hosted arm64 runner to some of my projects, which is how I discovered the need for this PR.

Oh, I missed that, I thought your PR was in response to a change I had missed where GitHub Actions offered ARM64 agents. Sorry.

Reply:
No worries!  I hope they start offering them soon.  I would greatly prefer not to need a self-hosted runner.